### ISSUE TITLE
Why opencv dependency?

### ISSUE THREAD

**Issue by miratcan** [knosmos/cmdpxl#13]
Just, why? 90 MB dependency on a simple keyboard-based pixel editor?

**Comment by knosmos** [comment]
That was the image manipulation library I was most familiar with when I made this project; feel free to make a fork that uses something like Pillow instead.

**Comment by therealmaxmoon** [comment]
I was looking for a tui image editor, because in general tui applications are really energy-efficient and small. 

While installing I got alarmed because of the huge download and it was downloading opencv and the first thing I thought was "This thing is spyware!", because I don't have an explanation why something like this needs opencv.

"Out of confidence" might be an explanation, even if it's a horrible excuse. 

@miratcan Have you found an alternative yet?

**Comment by miratcan** [comment]
@therealmaxmoon nope, but I think someone can build it with react + ink I think. https://github.com/vadimdemedes/ink
